micrometer-spring-legacy/src/main/java/io/micrometer/spring/autoconfigure/web/jetty/JettyMetricsAutoConfiguration.java

```diff
@@ -27,12 +33,17 @@
  * @author Michael Weirauch
  */
 @Configuration
-@ConditionalOnClass(name = "org.eclipse.jetty.server.Server")
-@ConditionalOnMissingClass("org.apache.catalina.startup.Tomcat")
```

Comment:
@mweirauch Any insight into why this was here?

Reply:
It used to be there before I last touched this. But you are right, @izeye has it - as usual - right ;)
Thanks for spotting!